src/Operation/Executable.php

```diff
@@ -31,8 +31,6 @@ interface Executable
 {
     /**
      * Execute the operation.
-     *
-     * @return mixed
      */
-    public function execute(Server $server);
+    public function execute(Server $server): mixed;
```

Comment:
Consider leaving this untyped or removing the interface entirely (that should warrant a PHPLIB ticket for our internal tracking).

Reply:
Removed for the moment and added tracking tickets.